static/app/views/alerts/wizard/index.tsx

```diff
@@ -54,7 +53,7 @@ class AlertWizard extends React.Component<Props, State> {
     const {organization, project, location} = this.props;
     const {alertOption} = this.state;
     const metricRuleTemplate = AlertWizardRuleTemplates[alertOption];
-    const isTransactionDataset = metricRuleTemplate?.dataset === Dataset.TRANSACTIONS;
+    const isMetricAlert = !!metricRuleTemplate?.dataset;
```

Comment:
nit: can just use `!!metricRuleTemplate`
```suggestion
    const isMetricAlert = !!metricRuleTemplate;
```